webpack/assets/javascripts/react_app/components/ChartBox/ChartBox.js

```diff
@@ -70,9 +66,7 @@ class ChartBox extends React.Component {
       bar: barChartProps,
     };
 
-    const panelChart = (
-      <Chart {...chartPropsForType[type]} config={this.props.config} />
```

Comment:
`props.config` was already destructed to `config`, using it, no other change here.